### ISSUE TITLE
NameError: name 'devlog' is not defined

### ISSUE THREAD

**Issue by joonis** [dashingsoft/pyarmor#1160]
I get this error when calling pyarmor with `--assert-call` or `--assert-import`.

```
$ pyarmor gen -i --assert-call -O ./build mypackage
INFO     Python 3.8.10
INFO     Pyarmor 8.1.1 (basic), xxxxxx, non-profits
INFO     Platform linux.x86_64
INFO     search inputs ...
INFO     find package at mypackage
INFO     find 1 top resources
CRITICAL unknown error, please check pyarmor.error.log
ERROR    name 'devlog' is not defined
```

pyarmor.error.log:
```
3364702 MainProcess 2023-04-15 18:27:24,888
Traceback (most recent call last):
  File "/home/xxx/.local/lib/python3.8/site-packages/pyarmor/cli/__main__.py", line 627, in main
    main_entry(sys.argv[1:])
  File "/home/xxx/.local/lib/python3.8/site-packages/pyarmor/cli/__main__.py", line 615, in main_entry
    args.func(ctx, args)
  File "/home/xxx/.local/lib/python3.8/site-packages/pyarmor/cli/__main__.py", line 179, in cmd_gen
    builder.process(options, pack=args.pack)
  File "/home/xxx/.local/lib/python3.8/site-packages/pyarmor/cli/generate.py", line 145, in process
    Pytransform3.pre_build(self.ctx)
  File "/home/xxx/.local/lib/python3.8/site-packages/pyarmor/cli/core/__init__.py", line 134, in pre_build
    return m.pre_build(ctx)
  File "<maker>", line 780, in pre_build
  File "<maker>", line 667, in build
  File "<maker>", line 3384, in process
  File "<maker>", line 2902, in rebuild
  File "<maker>", line 3073, in _get_module_types
  File "<maker>", line 2988, in prepare
  File "<maker>", line 2988, in prepare
  File "<maker>", line 2988, in prepare
  [Previous line repeated 1 more time]
  File "<maker>", line 2977, in prepare
  File "<maker>", line 2851, in add_node
NameError: name 'devlog' is not defined
```

**Comment by joonis** [comment]
Same with Pyarmor 8.1.5

**Comment by jondy** [comment]
Got it thanks, it will be fixed in next release 8.1.6

**Comment by jondy** [comment]
Fixed in 8.1.6
